Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -249,9 +255,10 @@ open class BasicFormat: SyntaxRewriter {
     case \ExpressionSegmentSyntax.backslash,
       \ExpressionSegmentSyntax.rightParen,
       \DeclNameArgumentSyntax.colon,
-      \StringLiteralExprSyntax.openQuote,
       \RegexLiteralExprSyntax.openSlash:
       return false
+    case \StringLiteralExprSyntax.openQuote:
+      return first?.tokenKind == .multilineStringQuote
```

Comment:
This seems a little odd to me. A multiline string quote should be followed by a newline, not just whitespace. I think we might need to introduce `requiresTrailingNewline` for that.